config/foreman-proxy-content.migrations/210415180240-enable-katello-agent.rb

```diff
@@ -0,0 +1,3 @@
+if answers['foreman_proxy_content'].is_a?(Hash) && answers['foreman_proxy_content']['enable_katello_agent'].nil?
+  answers['foreman_proxy_content']['enable_katello_agent'] = true
```

Comment:
We already had a parameter to enable/disable qpid dispatch. Should we instead "move" that answer?

Reply:
Yea, that's a good idea, as it allows a cleaner answers file that just lets the puppet module dictate the default for new installs and will handle if a user did actually disable qpid_router themselves.